lib/gitarro/backend.rb

```diff
@@ -63,105 +63,119 @@ def initialize(option = nil)
     @gbexec = TestExecutor.new(@options)
   end
 
-  # public method for get prs opens
-  # given a repo
-  def open_prs
-    prs = @client.pull_requests(@repo, state: 'open')
-    puts 'no Pull request OPEN on the REPO!' unless prs.any?
+  # Check if a PR list is empty
+  # If it is and we do want to consider the update time, return true
+  # Otherwise return false
+  def check_pr_list_empty(prs)
+    prs.empty? && @options[:changed_since] >= 0 ? true : false
+  end
+
+  # public method for get prs opened and matching the changed_since
+  # condition
+  def open_newer_prs
+    prs = []
+    @client.pull_requests(@repo, state: 'open').each do |pr|
+      prs << pr if pr_last_update_less_than(pr, @options[:changed_since])
```

Comment:
cosmetical : 

https://stackoverflow.com/questions/10569529/ruby-difference-between-array-and-arraypush
I tend to prefer prs.push(pr) (since it is more verbose) but `<<` it is ok

Reply:
Done, pending test and push.